go/base/tests/test_go_manage_contact_group.py

```diff
@@ -14,13 +14,9 @@ def setUp(self):
 
     def invoke_command(self, command, **kw):
         options = {
-            'email-address': self.user_helper.get_django_user().email,
-            'list': False,
-            'create': False,
-            'create-smart': False,
-            'delete': False,
+            'email_address': self.user_helper.get_django_user().email,
+            'command': [command]
```

Comment:
Please put trailing commas at the end of lists/dicts/whatever. It saves pain later when we reorder or add entries.